src/go/k8s/tests/e2e/node-select-tolerations/00-current-image.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: kubectl taint nodes kind-worker2 redpanda-node=true:NoSchedule
```

Comment:
hmm this will be problematic because if the test fails, we won't get the taint removed 🤔